routes/personal/schema.js

```diff
@@ -1,56 +1,74 @@
 /* istanbul ignore file */
 
-const isValidDate = require('../../utils/').isValidDate
+const isValidDate = require('../../utils/custom/validate.helpers.js').isValidDate
+const validateTelephoneFormat = require('../../utils/custom/validate.helpers.js').validateTelephoneFormat
+const validateDateFormat = require('../../utils/custom/validate.helpers.js').validateDateFormat
 
 const Schema = {
-  fullname: {
+  social: {
```

Comment:
SIN uses the Luhn Algorithm for validation https://www.npmjs.com/search?q=luhn if you want to implement a custom validation for it

Reply:
added